pkgs/development/python-modules/nitransforms/default.nix

```diff
@@ -13,14 +13,14 @@
 
 buildPythonPackage rec {
   pname = "nitransforms";
-  version = "24.1.2";
+  version = "25.0.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```

Comment:
```suggestion
  disabled = pythonOlder "3.10";
```

Reply:
@NickCao feel free to push this change? otherwise I can add it in a new PR after this is merged

Reply:
Pushed, also did some minor cleanups.